cmd/run.go

```diff
@@ -17,6 +17,7 @@ import (
 	"context"
 	"encoding/json"
 	"fmt"
+	"github.com/pingcap-incubator/tiup/pkg/repository/v0manifest"
```

Comment:
Please optimize all imports.

Reply:
is there an argument to goimports which will make it do this?

Reply:
I think `make fmt` will optimize the imports.

Reply:
It only organises them within groups :-(